### PR TITLE
Remove onlineFetch

### PR DESCRIPTION
**Pull request by nemaniarjun**
Tempeerory checking

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

File: uiplib/scrape/onlineFetch.py [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
-        """Initialize the fetch thread."""
-        Thread.__init__(self)
+        """Initialize the fetch."""
         self.url = url
         self.directory = directory
         self.count = count
 
     def run(self):
-        """Begin the download. Automatically called when a thread starts."""
+        """Begin the download. Automatically called when a Fetch starts."""
```
Comment (abhsag24) [inline uiplib/scrape/onlineFetch.py]: how does run get called? when its a  thread it gets called by `object.start()` but its not a thread anymore? youd want to change that :P

Comment (nemaniarjun) [inline uiplib/scrape/onlineFetch.py]: What if I add self.run() at line 14?

Comment (abhsag24) [inline uiplib/scrape/onlineFetch.py]: or merge it in the init itself? no point of making a new function then is there? you'd also want to remove the start() call in other files right?

Comment (nemaniarjun) [inline uiplib/scrape/onlineFetch.py]: Okay. Lets see :P

File: uiplib/scheduler.py [hunk 2]
```diff
@@ -94,14 +93,7 @@ def run(self):
             try:
                 thread_nos = len(self.website)
```
Comment (abhsag24) [inline uiplib/scheduler.py]: change variable name

**Comment by abhsag24** [comment]
@nemaniarjun also please add fexes tag in your commit i think we can merge this soon

**Comment by nemaniarjun** [comment]
Gonna fail coverage I think :p

**Pull request closed without merging**